### ISSUE TITLE
Learning rate bug (?)

### ISSUE THREAD

**Issue by aldakata** [huggingface/smollm#105]
I was checking the pretraining config files, and I saw that there is a 600k step LR  [as evidenced here](https://github.com/huggingface/smollm/blob/e5ead09dccc30626180a27dd2e654732f5d119a2/text/pretraining/smollm3/stage1_8T.yaml#L206C1-L206C27) decay stage during the first phase of pretraining, contrary to what the model card indicates:

Is this an unintended bug in the code? Was there LR decay during the first stage - as the config file suggests - or was there not?

Thank you for your time, and open-sourcing the whole project, great work 🫂 

<img width="2048" height="1229" alt="Image" src="https://private-user-images.githubusercontent.com/97572401/463773017-2bf61ea2-8d2e-426b-ba40-0242d34325d2.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTM4MDUyMTksIm5iZiI6MTc1MzgwNDkxOSwicGF0aCI6Ii85NzU3MjQwMS80NjM3NzMwMTctMmJmNjFlYTItOGQyZS00MjZiLWJhNDAtMDI0MmQzNDMyNWQyLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTA3MjklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwNzI5VDE2MDE1OVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWFjMzEzMzE3Zjg0NzFmZGQ3MWY2Yzc3ZjhmMGZkN2Y1ZjVmYzVlYzhhNDU4NDBkY2Q5ZTgwOGMzNmQ3NWE0OGImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.7A_Vp6IRAN0a1Pgc57wIWUHIMrIm3K3oSIEVbjlsXVc" />

**Comment by eliebak** [comment]
When we begin training, we don’t always know in advance how many steps the process will take. As a result, the learning rate schedule and training steps specified in the initial config may not reflect the final values. The final numbers for warmup, decay, and total training steps are the one in the final [config](https://github.com/huggingface/smollm/blob/main/text/pretraining/smollm3/stage3_9T_11T.yaml).

I agree this can be confusing for readers, so I’ll update the initial config to match the final training steps, thanks for pointing this out! 😊
